### PR TITLE
Add Documentation for Manual Changes after Code Generation

### PR DESCRIPTION
**Pull request by alzimmermsft**
# Description

Add a guide of all the manual changes that need to be done for certain code generation changes in `azure-search-documents`.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by jairmyree — APPROVED** [review]
LGTM!

File: sdk/search/azure-search-documents/swagger/README.md [hunk 1]
```diff
@@ -36,6 +36,40 @@ cd <swagger-folder>
 autorest --tag=searchindex
 autorest --tag=searchservice
 ```
+
+## Manual Changes
```
Comment (srnagar) [inline sdk/search/azure-search-documents/swagger/README.md:40]: Can some of these manual changes be done through customization?

**Review by srnagar — APPROVED** [review]
Thanks for documenting these manual changes needed for Search!

File: sdk/search/azure-search-documents/swagger/README.md (hunk 1 above)
Comment (alzimmermsft) [reply sdk/search/azure-search-documents/swagger/README.md:40]: I think so, short term is get this documentation out there and we can begin working it back by using code generation customizations to reduce the amount of code we have to maintain

**Pull request merged**